apps/desktop/src/bootstrap.ts

```
@@ -1,7 +1,13 @@
 // Copyright (c) 2026. 千诚. Licensed under GPL v3.
 
 import { appUpdateService } from '@services/AppUpdateService';
+import { completeManagedLogin, initializeManagedProviderState } from '@services/AuthService';
+import { createManagedSettingsFocusRequest, persistManagedSettingsFocusRequest } from '@services/AuthService/managedSettingsFocus';
```

ESLint

[error] Replace `·createManagedSettingsFocusRequest,·persistManagedSettingsFocusRequest·` with `⏎····createManagedSettingsFocusRequest,⏎····persistManagedSettingsFocusRequest,⏎`

(prettier/prettier)